lib/killbill_client/models/overdue.rb

```diff
@@ -8,24 +8,49 @@ class Overdue < OverdueAttributes
 
       class << self
 
-        def get_tenant_overdue_config(format, options = {})
+        def get_tenant_overdue_config_xml(options = {})
 
           require_multi_tenant_options!(options, "Retrieving an overdue config is only supported in multi-tenant mode")
 
-          get KILLBILL_API_OVERDUE_PREFIX,
+          get "#{KILLBILL_API_OVERDUE_PREFIX}/xml",
               {},
               {
-                  :head => {'Accept' => "application/#{format}"},
-                  :content_type => "application/#{format}",
+                  :head => {'Accept' => "text/xml"},
```

Comment:
I think for any `GET` operation we actually only need the `Accept` header and not the `content_type`. Also, maybe i got confused with the diff, bit why do we see `xml` on `Accept` and `json` on `content_type` ?

Reply:
I think if you can verify that only `Accept` header is required, then keep the second one.